test/support/shared_flows/minimum_wage_flow_test_helper.rb

```diff
@@ -0,0 +1,753 @@
+module MinimumWageFlowTestHelper
+  def test_shared_minimum_wage_flow_questions
+    test_question_what_would_you_like_to_check
+    test_question_are_you_an_apprentice
+    test_question_were_you_an_apprentice
+    test_question_how_old_are_you
+    test_question_how_old_were_you
+    test_question_how_often_do_you_get_paid
+    test_question_how_often_did_you_get_paid
+    test_question_how_many_hours_do_you_work
+    test_question_how_many_hours_did_you_work
+    test_question_how_much_are_you_paid_during_pay_period
+    test_question_how_much_were_you_paid_during_pay_period
+    test_question_is_provided_with_accommodation
+    test_question_was_provided_with_accommodation
+    test_question_current_accommodation_charge
+    test_question_past_accommodation_charge
+    test_question_current_accommodation_usage
+    test_question_past_accommodation_usage
+    test_question_does_employer_charge_for_job_requirements
+    test_question_did_employer_charge_for_job_requirements
+    test_question_current_additional_work_outside_shift
+    test_question_past_additional_work_outside_shift
+    test_question_current_paid_for_work_outside_shift
+    test_question_past_paid_for_work_outside_shift
```

Comment:
Yeah that seems a reasonable shout. 

I think I put them this way from initially thinking we'd call more of the methods in the flow, before I realised these were exactly the same.

Putting them altogether does have the downside of one monster-method rather than lots of followable ones, but does seem to cut out the annoyance of writing the question name about 4 times per file and, since there aren't other methods in the file, doesn't need much navigation.